jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -1546,6 +1546,28 @@ Function Start-Migration {
             new-item -ItemType Directory -Force -Path $jcAdmuTempPath 2>&1 | Write-Verbose
         }
         Write-ToLog -Message:($localComputerName + ' is currently Domain joined to ' + $WmiComputerSystem.Domain + ' NetBiosName is ' + $netBiosName)
+
+        # Get all schedule tasks that have State of "Ready" and not disabled and "Running"
+        $ScheduledTasks = Get-ScheduledTask | Where-Object { $_.TaskPath -notlike "*\Microsoft\Windows*" -and $_.State -ne "Disabled" -and $_.state -ne "Running" }
```

Comment:
Should we turn this into a function?

`Set-ScheduledTasks -op "Disable" -Tasks $scheduledTasks`
`Set-ScheduledTasks -op "Enable" -Tasks $scheduledTasks`

We have duplicated code in this PR so I'd suggest we move this into a function and determine the operation type with a parameter.

Reply:
+1 on this